tutorials/oauth/python/README.md

```diff
@@ -83,3 +83,55 @@ python consumer.py
 
     **Confluent-Kafka** -> To connect to EventHub using Kafka protocol
 
+
+## OAuth 2.0 Overview
+
+Event Hubs integrates with Azure Active Directory (Azure AD), which provides an OAuth 2.0 compliant authorization server. Azure role-based access control (Azure RBAC) can be used to grant permissions to Kafka client identities. 
+
+To grant access to an Event Hub resource, the security principal must be authenticated and an OAuth 2.0 token is returned. The token is then passed as part of the request to the Event Hub Service to authorize access to the resource. This is accomplished by setting the appropriate values in the Kafka client configuration. 
+
+For more information on the Azure AD OAuth 2.0 for Event Hubs see [Authorize access with Azure Active Directory](https://docs.microsoft.com/en-us/azure/event-hubs/authorize-access-azure-active-directory#overview)
+
+### Retrieve Azure Active Directory (AAD) Token
+
+The `DefaultAzureCredential` Class can be used to get a credential, and Kafka clients must use the scope of `https://<namespace>.servicebus.windows.net` to retrieve the access token from the Event Hub namespace. See [DefaultAzureCredential](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) for more information on this class. 
```

Comment:
Azure Docs say to use scope `https://<namespace>.servicebus.windows.net` but we have used `https://<namespace>.servicebus.windows.net/.default`. I wasn't sure  if either can be used or if something is missing in the docs. Is there any insight into this?

Reply:
Logged. Closing